### PR TITLE
Documention with markdown and remove mentions to master branch

### PR DESCRIPTION
**Pull request by olivroy**
Address #549 

* Redocument 
* Rename master -> main
* Add `@keywords internal` to defunct functions
* Convert to `md` documentation.
* Remove `@title` tags, and some `@description tags
* Reflow roxygen comments so the lines are not too wide.
* Remove one instance of superseded `tidyselect::one_of()` with `all_of()`.

Note: the style GHA doesn't work for me as I don't have write access. However, it fails. and recognizes that 10 files have to be changed.

### CODE REVIEW

File: R/adorn_rounding.R [hunk 1]
```diff
@@ -39,7 +39,7 @@
 #'
 #' cases %>%
 #'   adorn_percentages(, , ends_with("ed")) %>%
-#'   adorn_rounding(, , one_of(c("recovered", "died")))
```
Comment (olivroy) [inline R/adorn_rounding.R:42]: Unrelated, but one superseded thing I forgot in #547

**Comment by olivroy** [comment]
For the review,
you may want to focus on 92450eba105355d438b7573b1422cbe5fccb879b, 168c9de733cbcaf57d0151dc416a47b19492dd63 as those are the only manual changes I made except for conflict resolution) 

These where I removed some `@description` tags (but kept others) I also use Reflow comments to make sure that the roxygen comments were not too wide.

I think the overall diff is also pretty good to look at!

Many commits are a bit useless as I was just trying to resolve conflicts manually or messed my local branches (62c1d04dbc381aedb60dde3080b9df7771dce1b2, 73a20bb26e406b4f4bc87d5994080a0416603004, cc62e956f5a84d859643bc48786ca06a75461fea, 81476f383d48c24bd3ca550032e9b88aa75503f8)

The rest are automated changes initiated by styler, usethis, devtools or roxygen2md

File: R/remove_empties.R [hunk 2]
```diff
@@ -100,7 +100,7 @@ remove_empty <- function(dat, which = c("rows", "cols"), cutoff = 1, quiet = TRU
 #'   unique()
 #' @importFrom stats na.omit
 #' @family remove functions
-#' @seealso \code{\link[=remove_empty]{remove_empty()}} for removing empty
+#' @seealso [`remove_empty()`][remove_empty] for removing empty
```
Comment (billdenney) [inline R/remove_empties.R:103]: Is the second [remove_empty] needed here?

File: R/round_half_up.R [hunk 3]
```diff
@@ -1,12 +1,19 @@
-#' @title Round a numeric vector; halves will be rounded up, ala Microsoft Excel.
+#' Round a numeric vector; halves will be rounded up, ala Microsoft Excel.
 #'
 #' @description
```
Comment (billdenney) [inline R/round_half_up.R:3]: Can we drop this @description?

**Review by billdenney — CHANGES_REQUESTED** [review]
This looks very helpful!  I have a few minor comments on my read through.

File: R/remove_empties.R (hunk 2 above)
Comment (olivroy) [reply R/remove_empties.R:103]: No, it is just that before, it was a link to `remove_empty()`, but it showed, as `remove_empty`, for consistency, I agree we should change it.

File: man/round_half_up.Rd [hunk 4]
```diff
@@ -19,7 +19,8 @@ In base R \code{round()}, halves are rounded to even, e.g., 12.5 and
 11.5 are both rounded to 12.  This function rounds 12.5 to 13 (assuming
 \code{digits = 0}).  Negative halves are rounded away from zero, e.g., -0.5 is
 rounded to -1.
-
+}
+\details{
```
Comment (olivroy) [inline man/round_half_up.Rd:23]: Note this section moved to details

Comment (olivroy) [reply man/round_half_up.Rd]: Reverted

**Comment by olivroy** [comment]
@billdenney Thanks for the review. Your comments have been addressed. I added a commit last minute regarding `@keywords internal`. Details here https://github.com/sfirke/janitor/pull/551#issuecomment-1594856853  

I see you didn't have the change to review that, just wanted to flag that for transparency.

**Comment by billdenney** [comment]
I saw that one, and I appreciate the transparency.  😄

**Comment by olivroy** [comment]
Hi @sfirke, friendly reminder! 

whenever you have time, you can review this.

No user visible change overall.

Just some doc refactoring

Thanks

File: R/clean_names.R [hunk 5]
```diff
@@ -1,38 +1,38 @@
...
 #'
 #' Accented characters are transliterated to ASCII.  For example, an "o" with a
 #' German umlaut over it becomes "o", and the Spanish character "enye" becomes
 #' "n".
 #'
 #' This function takes and returns a data.frame, for ease of piping with
-#' \code{`\%>\%`}. For the underlying function that works on a character vector
-#' of names, see \code{\link[janitor]{make_clean_names}}.  \code{clean_names}
-#' relies on the versatile function \code{\link[snakecase]{to_any_case}}, which
+#' ``\%>\%``. For the underlying function that works on a character vector
```
Comment (sfirke) [inline R/clean_names.R:12]: Is this right, with two ticks to open and one tick to close?

File: R/statistical_tests.R [hunk 6]
```diff
@@ -1,17 +1,15 @@
-#' @title Apply stats::chisq.test to a two-way tabyl
+#' Apply `stats::chisq.test()` to a two-way tabyl
```
Comment (sfirke) [inline R/statistical_tests.R:1]: nice addition

File: R/statistical_tests.R [hunk 7]
```diff
@@ -121,7 +119,7 @@ chisq.test.tabyl <- function(x, tabyl_results = TRUE, ...) {
 
 
 
-#' @title Apply stats::fisher.test to a two-way tabyl
+#' Apply stats::fisher.test to a two-way tabyl
```
Comment (sfirke) [inline R/statistical_tests.R:122]: Do you want to make this `stats::fisher.test` to make it consistent with the formatting change above to `stats::chisq.test` ?

**Review by sfirke — COMMENTED** [review]
Looks great to me!  This is a great effort.  I left two small suggestions.

**Comment by sfirke** [comment]
hi @olivroy thanks for this nice cleanup job and sorry I took so long to review!  A few comments above, the changes will be <5 minutes.

File: R/statistical_tests.R (hunk 7 above)
Comment (olivroy) [reply R/statistical_tests.R:122]: I agree!

File: R/clean_names.R (hunk 5 above)
Comment (olivroy) [reply R/clean_names.R:12]: Corrected now 
`%>%` works

**Comment by olivroy** [comment]
Thanks for your kind review! 

Addressed in 24ef7bd

Minor additional tweaks pushed in 8418cb5

Styling in eb2c1ef

**Comment by sfirke** [comment]
Huzzah! Thanks again @olivroy 🙏

**Pull request merged**